ui/src/app/pages/data/data-set/new-dataset-modal.spec.tsx

```diff
@@ -11,7 +11,10 @@ import {
 import {waitOneTickAndUpdate} from 'testing/react-test-helpers';
 import {DataSetApiStub} from 'testing/stubs/data-set-api-stub';
 import {WorkspacesApiStub} from 'testing/stubs/workspaces-api-stub';
+import {DataSetExportRequest} from '../../../../generated/fetch';
```

Comment:
please make relative to application, not the current file (next line also violates this, but is less offensive and existing issue)